test/mocks/feed/post.input.dto.mock.ts

```diff
@@ -1,6 +1,6 @@
 import { PostCreateDto } from "../../../src/entities/feed/input/post.create.dto";
 
-const getDefault = (): PostCreateDto => ({
+const getDefaultCreateDto = (): PostCreateDto => ({
```

Comment:
aca se me empezo a complicar un poco entenderlo

Reply:
acá lo explica muy bien  [https://itnext.io/mocking-in-typescript-a97267f7cea9](url)